scripts/apps/publish/directives/SubscribersDirective.ts

```diff
@@ -284,6 +302,103 @@ export function SubscribersDirective(
             if (!$scope.subscribersList) {
                 fetchSubscribers();
             }
+
+            $scope.$watchGroup([
+                'subscriber.is_active',
+                'subscriber.schedule.startDate',
+                'subscriber.schedule.endDate',
+            ], () => {
+                $scope.validateScheduleDates();
+            });
+
+
+            $scope.updateStartDate = (date) => $scope.updateScheduleDate('startDate', date);
+            $scope.updateEndDate = (date) => $scope.updateScheduleDate('endDate', date);
+
+            /**
+            * Updates schedule dates and triggers validation
+            */
+            $scope.updateScheduleDate = function(key, value) {
+                if (!$scope.subscriber || !$scope.subscriber.schedule) return;
+
+                $scope.subscriber.schedule[key] = value;
+                $scope.saveEnabled = true;
+
+                $scope.$applyAsync(() => {
+                    $scope.validateScheduleDates();
+                });
+            };
+
+            /**
+            * Validates schedule dates against status of subscriber and inform the user
+            */
+            $scope.validateScheduleDates = function() {
+                if (!$scope.subscriber || !$scope.subscriber.schedule) return;
+
+                const {startDate, endDate} = $scope.subscriber.schedule;
+                const now = moment();
+
+                $scope.scheduleError.activeBeforeStartErrorMessage = null;
+                $scope.scheduleError.activeAfterEndErrorMessage = null;
+
+                const mStart = startDate ? moment(startDate) : null;
+                const mEnd = endDate ? moment(endDate) : null;
+
+                if ($scope.subscriber.is_active) {
+                    if (mStart && now.isBefore(mStart)) {
+                        $scope.scheduleError.activeBeforeStartErrorMessage = gettext(
+                            `Subscriber is currently active but will be deactivated until
+                            ${mStart.format('LLL')} when you save.`,
+                        );
+                    }
+
+                    if (mEnd && now.isAfter(mEnd)) {
+                        $scope.scheduleError.activeAfterEndErrorMessage = gettext(
+                            `Subscriber is currently active but the end date ${mEnd.format('LLL')} has passed. 
+                            It will be deactivated when you save.`,
+                        );
+                    }
+                } else if (
+                    mStart && mEnd &&
+                    now.isSameOrAfter(mStart) &&
+                    now.isSameOrBefore(mEnd)
+                ) {
+                    $scope.scheduleError.activeBeforeStartErrorMessage = gettext(
+                        `Subscriber is inactive but the current time is within the active schedule 
+                        (${mStart.format('LLL')} to ${mEnd.format('LLL')}). It will be activated when you save.`,
```

Comment:
do not use newlines in gettext - concatenate multiple strings using a `+` operator instead